ci/src/test/kotlin/com/saveourtool/test/CategoryEnum.kt

```diff
@@ -1,10 +1,9 @@
 package com.saveourtool.test
 
 enum class BenchmarkCategoryEnum {
```

Comment:
please add a note about that this enum should be synchronized with enum in save-cloud

Reply:
Added